corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -107,13 +108,40 @@ hqDefine('cloudcare/js/backbone/cases.js', function () {
     });
 
     cloudCareCases.caseViewMixin = {
-        lookupField: function (field) {
-            var parentCase;
+        lookupField: function (detailColumn) {
+            var parentCase,
+                field = detailColumn.field,
+                fieldValue;
             if (isParentField(field) && this.model.get('casedb')) {
                 parentCase = this.model.get('casedb').get(this.model.get('indices').parent.case_id);
-                return parentCase.getProperty(field.slice('parent/'.length));
+                fieldValue = parentCase.getProperty(field.slice('parent/'.length));
+            } else {
+                fieldValue = this.model.getProperty(field);
+            }
+            return this.formatDetailField(fieldValue, detailColumn);
+        },
+        formatDetailField: function(fieldValue, detailColumn) {
+            var agoInterval = detailColumn.time_ago_interval,
+                // See corehq/apps/app_manager/models#TimeAgoInterval
+                agoIntervalMap = {
```

Comment:
This is also defined in `detail-screen-config.js` (used in case-list config settings page). Not sure if there is an easy way to reference it though. (non-blocking)